static/js/message_edit.js

```diff
@@ -143,23 +143,31 @@ exports.show_topic_edit_spinner = function (row) {
     $(".topic_edit_cancel").hide();
 };
 
-exports.end_if_focused = function () {
-    const focused_elem = $(".message_edit").find(':focus');
-
-    if (focused_elem.length === 1) {
-        focused_elem.blur();
-        const row = focused_elem.closest('.message_row');
-        exports.end(row);
+exports.end_if_focused = function (from_inline_topic_edit) {
```

Comment:
~~~ js
exports.two_functions_disguised_as_one = (b) {
  if (b) {
    // do function1 stuff
  else {
    // do function2 stuff
  }
}

Reply:
It's fine to have two five-line functions here that are similar structurally; just try to keep them both nice and tidy.